step-templates/proxmox-stop-lxc.json

```diff
@@ -0,0 +1,104 @@
+    },
+    "Parameters": [
+      {
+        "Id": "9f1a3789-a4fe-4fc4-9a66-78e73d384248",
+        "Name": "Proxmox.Host",
+        "Label": "Proxmox Host",
+        "HelpText": "The hostname or IP address of the Proxmox cluster/host",
+        "DefaultValue": "1.2.3.4",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "240b1180-7b10-48ac-90fb-796fc8eaaf41",
+        "Name": "Proxmox.Port",
+        "Label": "Proxmox Port",
+        "HelpText": "Port number for Proxmox Cluster/Host",
+        "DefaultValue": "8006",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "37d5fad7-2bd3-45da-b30c-62116f0b1940",
+        "Name": "Proxmox.User",
+        "Label": "Proxmox User Account",
+        "HelpText": "The Proxmox user account associated with the api token.",
+        "DefaultValue": "root@pam",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "72a89f72-2571-435b-b0f7-a383f08572b9",
+        "Name": "Proxmox.Node",
+        "Label": "Proxmox Node",
+        "HelpText": "The Proxmox node in the cluster.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "b33b88c3-3949-4563-91a2-4ae264bc2d7f",
+        "Name": "Proxmox.TokenID",
+        "Label": "Proxmox Token ID",
+        "HelpText": "This is token id that was used to create an API token in proxmox.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "eff9ca9b-b3ea-44e9-aa79-489d1d9161ee",
+        "Name": "Proxmox.Token",
+        "Label": "Proxmox API Token",
+        "HelpText": "The API Token secret key",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Sensitive"
+        }
+      },
+      {
+        "Id": "f7be6495-cc3b-4492-bb1b-e8a178a36a91",
+        "Name": "Proxmox.LXC.VMID",
+        "Label": "LXC VM ID",
+        "HelpText": "The LXC VMID you want to shutdown",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "94de0cbc-b722-4788-858f-fb2b32b57fd8",
+        "Name": "Proxmox.LXC.Stop",
+        "Label": "Force Stop LXC",

```

Comment:
Is this label accurate? The code implies if this is false, then it will shutdown, not force a stop?

Reply:
If this parameter is enabled it will Force stop the LXC Container instead of gracefully shutting down

Reply:
Updated help text to better describe the parameter.